exporters/metrics/src/main/java/com/google/cloud/opentelemetry/metric/MetricTranslator.java

```diff
@@ -48,7 +49,6 @@ public final class MetricTranslator {
 
   private static final Logger logger = LoggerFactory.getLogger(MetricTranslator.class);
 
-  static final String DESCRIPTOR_TYPE_URL = "workload.googleapis.com/";
```

Comment:
Instead of removing this, we could move this to `MetricConfiguration.java` since this is used in multiple places in the code and has a special meaning (default prefix).